invesalius/data/brainmesh_handler.py

```diff
@@ -66,7 +63,7 @@ def __init__(self, img_path, mask_path, n_peels, affine_vtk):
         self.refImageSpace2_xyz.SetTransform(refImageSpace2_xyz_transform)
 
         xyz2_refImageSpace_transform = vtk.vtkTransform()
-        qFormMatrix.Invert()
+        # qFormMatrix.Invert()
```

Comment:
Here, I'd have a similar question: would it benefit the project to adopt the stylistic principle to not have commented out code in the codebase?

Here are a few references that instruct to avoid commenting out code:

https://kentcdodds.com/blog/please-dont-commit-commented-out-code

https://agiletribe.wordpress.com/2015/12/26/never-leave-commented-code-in-the-source/

https://www.nayuki.io/page/dont-share-commented-out-code

Again, I'm not sure how to proceed with this proposal, but maybe the main authors of the project could comment on it.

Reply:
It's a work in progress yet, I will remove the commented codes. Also, there are other code needed anymore, like the transformations. I just submitted this as PR because it simpler to others test it.

Reply:
Ah, my apologies, I didn't realize that. Consider my comment retracted!